scripts/manage_porkbun.py

```
@@ -30,6 +43,167 @@
+    lines = [f"\n# catchup {date_str}\n"]
+
+    if is_template and diff.porkbun_only:
+        lines.append("# TODO: replace raw IPs with template vars if applicable\n")
+
+    for rec in diff.porkbun_only:
+        prio_part = f"    {rec['prio']}" if rec.get("prio") and rec["prio"] not in ("None", "0") else ""
+        lines.append(f"{rec['type']:<6}  {rec['name']:<45}  {rec['content']:<40}  {rec['ttl']}{prio_part}\n")
+
+    for pb_rec, cfg_rec in diff.value_conflicts:
+        prio_part = f"    {cfg_rec['prio']}" if cfg_rec.get("prio") and cfg_rec["prio"] not in ("None", "0") else ""
+        lines.append(f"# CONFLICT: choose one and remove the other\n")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)